interfaces/builtin/ppp.go

```diff
@@ -55,6 +55,11 @@ var pppConnectedPlugKmod = []string{
 	"ppp_generic",
 }
 
+const pppConnectedPlugUDev = `
+KERNEL=="ppp", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="tty[A-Z]*", TAG+="###SLOT_SECURITY_TAGS###"
```

Comment:
We're using this elsewhere:
```
KERNEL=="tty[A-Z]*[0-9]*", TAG+="###SLOT_SECURITY_TAGS###"
```

Also, s/###SLOT_SECURITY_TAGS###/###CONNECTED_SECURITY_TAGS###/